hoodie-utilities/src/main/java/com/uber/hoodie/utilities/UtilHelpers.java

```diff
@@ -131,8 +133,8 @@ public static String parseSchema(FileSystem fs, String schemaFile) throws Except
     return new String(buf.array());
   }
 
-  private static SparkConf buildSparkConf(String appName, String defaultMaster) {
-    SparkConf sparkConf = new SparkConf().setAppName(appName);
+  private static SparkConf buildSparkConf(String appName, String defaultMaster, Map<String, String> additionalConfigs) {
```

Comment:
have an overloaded method instead with just takes 2 params and passes down an empty map?

Reply:
Done